test/integration/versionchecker/versionchecker_test.go

```diff
@@ -48,17 +49,29 @@ func loadManifests() (io.Reader, error, func() (string, error), func()) {
 	}
 	fileReader := tar.NewReader(data)
 
-	return fileReader, nil, func() (string, error) {
+	nextFunc := func() (string, error) {
+		for {
```

Comment:
note: The changes in this file mostly reflect that our way of tarring up files stores them in the tar file with names like `./example` rather than bazel's way which looks like `example`. I think this test should handle either format since both formats are totally valid in a given tar file.